speakers.js

```diff
@@ -0,0 +1,169 @@
+col7.appendChild(p3);
+col7.appendChild(hrSpeaker2);
+col7.appendChild(p4);
+
+row4.appendChild(col6);
+row4.appendChild(col7);
+col5.appendChild(row4);
+row2.appendChild(col5);
+
+const col8 = document.createElement('div');
+col8.classList.add('col-12', 'col-md-6', 'd-none', 'd-md-block');
+const row5 = document.createElement('div');
+row5.classList.add('row', 'mb-5', 'p-4', 'bg-white');
+const col9 = document.createElement('div');
+col9.classList.add('col-3', 'speaker-block', 'pr-0', 'pl-0');
+const img3 = document.createElement('img');
+img3.classList.add('img-fluid', 'rounded', 'speaker-avatar');
+img3.src = './images/speaker1.jpeg';
+img3.alt = 'Speaker';
+col9.appendChild(img3);
+
+const col10 = document.createElement('div');
+col10.classList.add('col-9', 'speaker-block', 'pr-0', 'pl-4');
+const h3Speaker3 = document.createElement('h3');
+h3Speaker3.classList.add('mb-1', 'section-title', 'font-weight-bold');
+h3Speaker3.textContent = 'Jane Doe';
+const p5 = document.createElement('p');
+p5.classList.add('red', 'font-italic');
+p5.textContent = 'Full Stack Web Developer';
+const hrSpeaker3 = document.createElement('hr');
+hrSpeaker3.classList.add('hr-speakers', 'ml-0', 'mb-3');
+const p6 = document.createElement('p');
+p6.textContent = 'Jane is a highly skilled front-end developer with a passion for creating visually stunning and user-friendly websites. With 5 years of experience, she is adept in technologies such as HTML, CSS, and JavaScript';
+
+col10.appendChild(h3Speaker3);
+col10.appendChild(p5);
+col10.appendChild(hrSpeaker3);
+col10.appendChild(p6);
+
+row5.appendChild(col9);
+row5.appendChild(col10);
+col8.appendChild(row5);
+row2.appendChild(col8);
+
+const col11 = document.createElement('div');
+col11.classList.add('col-12', 'col-md-6', 'd-none', 'd-md-block');
+const row6 = document.createElement('div');
+row6.classList.add('row', 'mb-5', 'p-4', 'bg-white');
+const col12 = document.createElement('div');
+col12.classList.add('col-3', 'speaker-block', 'pr-0', 'pl-0');
+const img4 = document.createElement('img');
+img4.classList.add('img-fluid', 'rounded', 'speaker-avatar');
+img4.src = './images/speaker2.jpeg';
+img4.alt = 'Speaker';
+col12.appendChild(img4);
+
+const col13 = document.createElement('div');
+col13.classList.add('col-9', 'speaker-block', 'pr-0', 'pl-4');
+const h3Speaker4 = document.createElement('h3');
+h3Speaker4.classList.add('mb-1', 'section-title', 'font-weight-bold');
+h3Speaker4.textContent = 'Jane';
+const p7 = document.createElement('p');
+p7.classList.add('red', 'font-italic');
+p7.textContent = 'Mobile App Developer';
+const hrSpeaker4 = document.createElement('hr');
+hrSpeaker4.classList.add('hr-speakers', 'ml-0', 'mb-3');
+const p8 = document.createElement('p');
+p8.textContent = 'Samantha is a highly skilled mobile app developer with a strong background in both iOS and Android development. With 4 years of experience, she is able to create high-performance and user-friendly mobile applications.';
```

Comment:
On mobile, the rest of the speakers didn't show up, find a screenshot from your webpage below;
<img width="496" alt="image" src="https://user-images.githubusercontent.com/31897434/215145412-acf1fc68-87a6-484f-9e79-32517f083c3d.png">

Make sure the user can see all the speakers.

Reply:
@hafedEfheij kindly address this request change.